errors/no-router-instance.md

```diff
@@ -2,8 +2,13 @@
 
 #### Why This Error Occurred
 
+Next.js is universal, which means it executes code first server-side with NodeJS, then on the client-side, `window` object is not defined in NodeJS, so some methods are not supported or availale at build time.
```

Comment:
```suggestion
Next.js is universal, which means it executes code first server-side with NodeJS, then on the client-side, `window` object is not defined in NodeJS, so some methods are not supported or available at build time.
```

The error document is unrelated to `window` not being available 🤔

Reply:
Thanks

Reply:
I have fixed the typo